docs/topics/settings.rst

```diff
@@ -418,6 +418,36 @@ This setting also affects :setting:`DOWNLOAD_DELAY` and
 :ref:`topics-autothrottle`: if :setting:`CONCURRENT_REQUESTS_PER_IP`
 is non-zero, download delay is enforced per IP, not per domain.
 
+.. setting:: DEFAULT_DROPITEM_LOG_LEVEL
+
+DEFAULT_DROPITEM_LOG_LEVEL
+--------------------------
+
+Default: ``'WARNING'``
+
+This setting allows you to configure the default log severity level for dropped items in Scrapy. When an item is dropped using the ``DropItem`` exception, the severity level of the corresponding log message will be determined by this setting, unless a specific severity is explicitly provided in the exception.
```

Comment:
``` ``DropItem`` ``` → ``` :exc:`scrapy.exceptions.DropItem` ```

Also, it might be useful to mention item pipelines (with a ref pointing to their docs) as an example of a Scrapy component that drops an item.